docs/Style-Guide.md

```diff
@@ -95,10 +94,10 @@ Condescending terms are words that include:
 * Just 
 * Easy
 * Simply
-* Basically
-* Obviously
+* boogeyman 
+* His/He
```

Comment:
Why did these change? The original two are correct. These are not.

Reply:
Alright.